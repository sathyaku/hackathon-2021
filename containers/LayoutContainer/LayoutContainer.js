import React from 'react';
import layOutStyle from './LayoutContainer.style';
import ContainerWrap from '@/ui/Container';

const LayoutContainer = ({ products }) => {
  const { results = [] } = products;
  return (
    <ContainerWrap>
      <style jsx>{layOutStyle}</style>
    </ContainerWrap>
  );
};

export default LayoutContainer;
