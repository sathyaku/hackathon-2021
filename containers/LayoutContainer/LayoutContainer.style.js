import css from 'styled-jsx/css';

const backgroundColor = '#eee';

const layOutStyle = css`
  $color: red;
  .container {
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: flex-start;
    flex-wrap: wrap;
    width: 100%;
    margin: 100px auto;

    @mixin desktop {
      max-width: 1150px;
    }
  }
`;

export default layOutStyle;
