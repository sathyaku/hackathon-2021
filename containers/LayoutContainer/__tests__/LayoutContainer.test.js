import React from 'react';
import { mount } from 'enzyme';

import LayoutContainer from '../LayoutContainer';

describe('Title Component', () => {
  it('should render without crashing', () => {
    const wrapper = mount(<LayoutContainer />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
