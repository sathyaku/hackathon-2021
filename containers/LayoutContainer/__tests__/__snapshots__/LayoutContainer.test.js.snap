// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component should render without crashing 1`] = `
"<LayoutContainer>
  <div className=\\"jsx-678447535 hello\\">
    <p className=\\"jsx-678447535\\">
      Hello World
    </p>
    <a href=\\"/about\\" className=\\"jsx-678447535\\">
       go to about 
    </a>
    <JSXStyle id=\\"678447535\\" />
  </div>
</LayoutContainer>"
`;
